package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Please add a `start script` to start the app instead of using the `localhost`.

Reply:
how ?

Reply:
@dasileker  can use please tell me more about that

Reply:
Sorry to respond late, you can just create a const that  has an array of object inside and populate the const into that specific div by calling the key-value of it